README.md

```diff
@@ -300,7 +310,22 @@ If you cannot find suitable test data on this repository, please contact us on t
       * fastq
         * 'test.fastq.gz' synthetic raw long-read sequencing reads of the genome of the mammalian-gut-residing _Bacteroides fragilis_ YCH46 bacterium (NC_006347). Originally generated for the [MAG pipeline test dataset](https://github.com/nf-core/test-datasets/tree/mag).
 
-## generic
+### pangenomics
+
+* homo_sapines
+  * pangenome
+    * 'pangenome.fa': A FASTA file which contains several related genomes for the input of `wfmash`.
```

Comment:
```suggestion
    * 'pangenome.fa': A FASTA file which contains several related genomes.
```
i guess other tools could use this too, right?

Reply:
I guess some of them. That depends on their downstream way of handling things.